src/main/scala/device/standalone/StandAloneDebugModule.scala

```diff
@@ -45,12 +47,16 @@ class StandAloneDebugModule (
 
   // interrupts
   val debugModuleIntNode = IntSinkNode(IntSinkPortSimple(hartNum, 1))
-  debugModuleIntNode :*= debugModule.debug.dmOuter.dmOuter.intnode
+  debugModuleIntNode :*= IntBuffer() :*= debugModule.debug.dmOuter.dmOuter.intnode
   val int = InModuleBody(debugModuleIntNode.makeIOs())
 
   class StandAloneDebugModuleImp(val outer: StandAloneDebugModule)(implicit p: Parameters) extends StandAloneDeviceImp(outer) {
     val io = IO(new outer.debugModule.DebugModuleIO)
     io <> outer.debugModule.module.io
+    outer.debugModule.module.io.resetCtrl.hartIsInReset :=
```

Comment:
`hartIsInReset` should be long enough, so that DebugModule in slower clock domain can sample at least one signal.